spanner/integration_test.go

```diff
@@ -356,7 +387,7 @@ const (
 func TestMain(m *testing.M) {
 	cleanup := initIntegrationTests()
 	defer cleanup()
-	for _, dialect := range []adminpb.DatabaseDialect{adminpb.DatabaseDialect_GOOGLE_STANDARD_SQL, adminpb.DatabaseDialect_POSTGRESQL} {
```

Comment:
@rahul2393 
I think this was committed unknowingly.
Can we add back `adminpb.DatabaseDialect_GOOGLE_STANDARD_SQL` as part of dialect tests?